packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -101,6 +101,30 @@ const EVENTS = {
     group: GROUPS.rp,
     event: 'engage',
   },
+  'flow.email-domain-validation.engage': {
+    group: GROUPS.registration,
+    event: 'domain_validation_engage',
```

Comment:
From https://github.com/mozilla/fxa/issues/3196#issuecomment-556473355, should this be `triggered` rather than `engage`?